packages/core-data/src/components/Button.js

```diff
@@ -46,9 +51,10 @@ const Button = (props: Props) => (
       { 'px-4': !props.icon },
       { 'rounded-full px-2': props.icon },
       { 'bg-secondary': props.secondary },
-      { 'border border-solid border-gray-200': !props.primary },
-      { 'bg-primary hover:bg-primary hover:saturate-50 text-white fill-white': props.primary },
+      { 'border border-solid border-gray-200 bg-white': !props.primary },
```

Comment:
Will this conflict with the `bg-secondary` class?

Reply:
Good catch!